HW2/index.css

```diff
@@ -0,0 +1,237 @@
+* {
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+}
+body {
+	background-color: #f9f9f9;
+}
+.wrapper {
+	width: 80%;
+	margin: 0 auto;
+	background-color: #fff;
+	box-shadow: 0 0 10px rgba(0,0,0,0.5);
+	padding: 50px;
+}
+/****************HEADER**************/
+
+.menu {
+	background: -moz-linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* ff3.6+ */
+	background: -webkit-gradient(linear, left bottom, right top, color-stop(0%, rgba(69,68,68,1)), color-stop(100%, rgba(94,94,94,1))); /* safari4+,chrome */
+	background: -webkit-linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* safari5.1+,chrome10+ */
+    background: -o-linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* opera 11.10+ */
+    background: -ms-linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* ie10+ */
+    background: linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* w3c */
+    filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#5e5e5e', endColorstr='#454444',GradientType=1 ); /* ie6-9 */
+	margin-top: 40px;
+	display: flex;
+	background-color: #ffffff;
+    background-image: linear-gradient(to top, #444343 0%, #5e5e5e 100%);
+    list-style: none;
+
+}
+
+.menu__link {
+	color: #fff;
+	text-decoration: none;
+	display: block;
+	height: 40px;
+	padding: 10px 25px 25px;
+	margin-right: 15px;
+	margin-left: 10px;
+	text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.5);
+}
+.menu__link:first-child {
+	margin-left: 0px;
+}
+.menu__link:hover {
+	background-color: #353535;
+	border-right: 1px solid #838383;
+	border-left: 1px solid #838383;
+	margin-right: 14px;
+	margin-left: 9px;
+}
+.menu__link:first-child:hover {
```

Comment:
WTF? Do you know that each link is the first child in it is only li? It is the only child. Rewrite this